.gitignore

```diff
@@ -56,3 +56,6 @@ typings/
 
 # dotenv environment variables file
 .env
+
+# Removed IDEA folder
```

Comment:
This should describe what the file is

Reply:
Done https://github.com/foundersandcoders/express-handlebars-workshop/pull/11/commits/8872318659cf56a5cc50302f427f693e1ae7f5e4